Makefile

```diff
@@ -325,7 +325,3 @@ help:  ## Show Help menu
 
 # helpers
 gen: gen-ui gen-api clients gen-docs
-
-delta-plugin: ## Build delta plugin
```

Comment:
If you don't remove the plugin itself, you can keep this one for those that wish to build it...

Reply:
I'll open another PR to remove the code - thanks!